shipyard_blueprints/sftp/shipyard_sftp/cli/move_file.py

```diff
@@ -41,177 +46,78 @@ def get_args():
-            k = paramiko.RSAKey.from_private_key_file(key)
-            ssh.connect(hostname=host, port=port, username=username, pkey=k)
-            return ssh.open_sftp()
-        else:
-            transport = paramiko.Transport((host, int(port)))
-            transport.connect(None, username, password)
-            return paramiko.SFTPClient.from_transport(transport)
-    except Exception as e:
-        print(e)
-        print(
-            f"Error accessing the SFTP server with the specified credentials"
-            f" {host}:{port} {username}:{key}"
+        source_file_name = args.source_file_name
+        source_folder_name = args.source_folder_name
+        source_full_path = shipyard.combine_folder_and_file_name(
+            source_folder_name, source_file_name
         )
-        sys.exit(ec.EXIT_CODE_INCORRECT_CREDENTIALS)
-
-
-def main():
-    args = get_args()
-    host = args.host
-    port = args.port
-    username = args.username
-    password = args.password
-    key = args.key
-    if not password and not key:
-        print("Must specify a password or a RSA key")
-        return
-
-    key_path = None
-    if key:
-        if not os.path.isfile(key):
-            fd, key_path = tempfile.mkstemp()
-            print(f"Storing RSA temporarily at {key_path}")
-            with os.fdopen(fd, "w") as tmp:
-                tmp.write(key)
-            key = key_path
-        client = get_client(host=host, port=port, username=username, key=key)
-    elif password:
-        client = get_client(host=host, port=port, username=username, password=password)
-
-    source_file_name = args.source_file_name
-    source_folder_name = args.source_folder_name
-    source_full_path = shipyard.files.combine_folder_and_file_name(
-        source_folder_name, source_file_name
-    )
-    destination_folder_name = shipyard.files.clean_folder_name(
-        args.destination_folder_name
-    )
-    source_file_name_match_type = args.source_file_name_match_type
-
-    ## check to see if the destination folder exists, if not a directory will be created
-    if destination_folder_name and not folder_exists(client, destination_folder_name):
-        client.mkdir(destination_folder_name)
-
-    if source_file_name_match_type == "regex_match":
-        file_names = find_sftp_file_names(client, prefix=source_folder_name)
-        matching_file_names = shipyard.files.find_all_file_matches(
-            file_names, re.compile(source_file_name)
+        destination_folder_name = shipyard.clean_folder_name(
+            args.destination_folder_name
         )
-        print(f"{len(matching_file_names)} files found. Preparing to move...")
+        source_file_name_match_type = args.source_file_name_match_type
 
-        for index, key_name in enumerate(matching_file_names, 1):
-            destination_full_path = shipyard.files.determine_destination_full_path(
+        if source_file_name_match_type in [
+            "exact_match",
+            "",
```

Comment:
I don't think the "" is necessary, given the condition:
```    parser.add_argument(
        "--source-file-name-match-type",
        dest="source_file_name_match_type",
        choices={"exact_match", "regex_match"},
        default="exact_match",
        required=False,
        ```

Reply:
That is to handle how the application sends empty values. Essentially enforcing the default value

Reply:
It's fine to leave it in, but in the application this would be a `Select` value and those always have a default option